autoPyTorch/pipeline/components/setup/network_backbone/ResNetBackbone.py

```diff
@@ -320,6 +329,7 @@ def forward(self, x: torch.FloatTensor) -> torch.FloatTensor:
             if self.config["use_skip_connection"]:
                 residual = self.shortcut(x)
 
+        # TODO make the below code better
```

Comment:
maybe you can also add the fix in line no. 256 which we discussed offline.

Reply:
I actually noticed that it is correct, if you look at https://arxiv.org/pdf/1705.07485.pdf `Page 2, Figure 1`, to the left it actually has the residual connection still flowing (Same for https://arxiv.org/pdf/1802.02375.pdf, `Page 3 Figure 1`. It confused me initially for `shake-shake` and `shake-drop` since I only looked at the right.  So, the residual connection can be applied for all 3 and it is correct in my pov.

Reply:
okay thanks for the explanation